theme.json

```diff
@@ -545,7 +539,7 @@
 			}
 		},
 		"spacing": {
-			"blockGap": "var:custom|spacing|small",
+			"blockGap": "var(--wp--preset--spacing--30, clamp(1.5rem, 5vw, 2rem))",
```

Comment:
Do you think we need the fallback here? Core provides one as well; not sure if we need to double up. What do you think?

Reply:
This fallback is for backward compatibility. 
WP 6.0 does not have support step spacing. The core did not provide one when I tested it without a fallback.

Reply:
I'm pretty sure these are added as a fallback in 6.1+

<img width="310" alt="CleanShot 2022-10-05 at 15 13 35@2x" src="https://user-images.githubusercontent.com/1813435/194143262-ec45da6f-8a7e-44d5-9f80-5e8d04e5851c.png">

Reply:
That's with TT2 theme active, which doesn't have spacing scale.

Reply:
> I'm pretty sure these are added as a fallback in 6.1+

Yes, these are added as a fallback for 6.1+. 
I am worried about 6.0 to 6.0.2. I added(`clamp(1.5rem, 5vw, 2rem)`) a fallback for 6.0 to 6.0.2. 


> That's with TT2 theme active, which doesn't have spacing scale.

Did you test it without the Gutenberg plugin?

Reply:
Yes I did test w/o G.

The fallback is for pre 6.1 yea? So if you're using Extendable on <6.1, the CSS fallback should trigger? I'm just not following 100% on what the clamp() is for, if we have the CSS fallback as well.

Reply:
The current CSS fallback is only for custom spacing var such as --wp--custom--spacing--small/medium/large. 

I have come to learn we should use static spacing for the `blockGap` rather than responsive. Because on small devices, `blockGap` responsive spacing becomes too small, and it makes it hard to see a separation between two blocks.

![image](https://user-images.githubusercontent.com/32260742/194653796-83d31022-c008-4f3b-aa52-093af5c855eb.png)